src/ark/phenotyping/post_cluster_utils.py

```diff
@@ -97,6 +98,13 @@ def create_mantis_project(cell_table, fovs, seg_dir, pop_col,
     # generate unique numeric value for each population
     small_table['pop_vals'] = pd.factorize(small_table[pop_col].tolist())[0] + 1
 
+    ccmd_pop = data_utils.CellClusterMaskData(
+        data=small_table,
+        fov_col=settings.FOV_ID,
+        label_col=settings.CELL_LABEL,
+        cluster_col="pop_vals",
+    )
+
```

Comment:
Can we have `fov_col` and `label_col` be arguments to the `create_mantis_project` function and have the default values be `settings.FOV_ID` and `settings.CELL_LABEL`? Not a huge change but I found before when testing that it was helpful to be able to change these values directly from the notebook (for example, if I'm using a different cell table or something to not have to change the settings file)